Google.PowerShell/Google.PowerShell.csproj

```diff
@@ -23,6 +23,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Google.PowerShell.XML</DocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
Please add an XML doc comment about what the warning is. As looking at the file you don't know if 1591 is important or not.